.gitignore

```diff
@@ -47,3 +47,4 @@ build/Release
 node_modules
 yarn.lock
 
+kerberos
```

Comment:
what's motivating this change? AFAICT nothing about the driver makes a `kerberos` folder in the source tree, did you do this locally?

Reply:
The README makes it seem like it needed to be installed at source.

https://github.com/mongodb/node-mongodb-native#diagnosing-on-unix